alts/src/test/java/io/grpc/alts/internal/AltsAuthContextTest.java

```diff
@@ -36,6 +40,8 @@
   private static final String TEST_PEER_SERVICE_ACCOUNT = "peer@gserviceaccount.com";
   private static final String TEST_RECORD_PROTOCOL = "ALTSRP_GCM_AES128";
 
+  private static final Map<String, String> TEST_PEER_ATTRIBUTES = new HashMap<String, String>();
```

Comment:
Can we use a local variable instead of a static variable?

Reply:
I was not sure so I just made it static since that is what the other sample data was to test the other methods. Do you think I should make it local?

Reply:
Yeah I think so. IMO, the reason why we want to make some test data, like TEST_RECORD_PROTOCOL, static is because these strings might be used multiple times in a place, and it's better to use a `static final` string to refer them, to avoid misspelling,  etc. This is similar to a `const string` in C++.
Map doesn't have this issue, so we'd better keep it local. However, we could make the key and value static. See my inline comments for more.